docs/dg/tools.md

```diff
@@ -0,0 +1,64 @@
+<variable name="title">Appendix: Tools for front-end development</variable>
```

Comment:
For this page:
- `html` should be changed to HTML
- `pug` should be changed to Pug
- `scss` should be changed to SCSS
- `css` should be changed to CSS
- `JavaScript` should be changed to JavaScript
- `Vue.js` should be changed to Vue.js
- `Vuex` should be changed to Vuex

I think that the names of the programming languages and frameworks don't need to be enclosed in code tags and can be written normally.